src/core/loaders/Loader.ts

```diff
@@ -258,6 +265,18 @@ export abstract class Loader extends Disposable {
       return this.getShadowNodeByKey(key)
     if (node && node.type !== 'tree')
       return node
+
+    // TODO: maybe use something other than Config.displayLanguage?
+    if (
+      node
+      && node.type === 'tree'
+      && Config._keyStyle !== 'flat'
+      && this.treeNodeValueHasPluralizationKeys(node.values[Config.displayLanguage])
+    ) {
+      const subkey = this.firstPluralizationKey(node.values[Config.displayLanguage])
```

Comment:
Would it work to return `node` here, so "Go to Definition" lands on the parent pluralization key, or does that cause problems in other parts of the code?